src/content/en/fundamentals/performance/audit/index.md

```diff
@@ -81,20 +85,24 @@ If nothing else, simply use [Lighthouse](/web/tools/lighthouse/) to check for:
 
   * HTTPS: [every site should deliver all assets over HTTPS](/web/fundamentals/security/encrypt-in-transit/why-https).
   * Server settings: your web server or CDN should [use compression correctly](/web/fundamentals/performance/optimizing-content-efficiency/optimize-encoding-and-transfer), [use HTTP/2](/web/fundamentals/performance/http2/), and [include appropriate headers](/web/fundamentals/performance/optimizing-content-efficiency/http-caching) to enable your browser to cache resources.
-  * Script elements that can be moved to the bottom of the page and/or given an [async or defer](http://peter.sh/experiments/asynchronous-and-deferred-javascript-execution-explained/) attribute.
+  * Script elements that can be moved to the bottom of the page and/or given 
+an [async or defer](http://peter.sh/experiments/asynchronous-and-deferred-javascript-execution-explained/){: .external} attribute.
```

Comment:
lines should wrap at 80 chars

Reply:
Won't this break the markup if I wrap at 80 characters?

Reply:
Nope, works fine.

Reply:
Changed.

Reply:
Adding line breaks to link markup seems to not work:

![image](https://user-images.githubusercontent.com/205226/53238169-81a86c80-3690-11e9-9665-47a846501f19.png)

Is there some other way to do this?

Reply:
Looks like you figured it out.